cmake/reports/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if (CONFIG_BUILD_WITH_TFM)
       -o ${CMAKE_BINARY_DIR}
       ${workspace_arg}
       -d ${report_depth}
+      --prefix tfm
```

Comment:
`size_report` already supports writing to an alternative file using `--json <file>`, so why not use what we have instead of introducing yet another argument ?

```suggestion
      --json tfm_${report}.json
```

Reply:
Fixed